configure.ac

```diff
@@ -9265,6 +9265,14 @@ then
     AM_CFLAGS="$AM_CFLAGS -DALT_ECC_SIZE"
 fi
 
+AC_ARG_ENABLE([default-devid],
```

Comment:
I almost think this should be an option on `--enable-cryptocb` such as `--enable-cryptocb=default` or `--enable-cryptocb=no-default-devid`. Someone first picking up wolfssl and running `./configure -h ` might not know what a devid is or that it is related to cryptocb.

Reply:
I agree. Something like `--enable-cryptocb=no-default-devid`.